internal/provider/resource_tfe_variable_test.go

```diff
@@ -725,6 +725,7 @@ resource "tfe_organization" "foobar" {
 resource "tfe_workspace" "foobar" {
   name         = "workspace-test"
   organization = tfe_organization.foobar.id
+  execution_mode = "remote"
```

Comment:
This test uses an old provider version for the first step, so applying with the modern provider will produce a diff because the `setting_overwrites` need to be set in order for the provider to know that it is communicating with a version of TFE that supports `setting_overwrites`. 

Explicitly setting the `execution_mode` eliminates this diff, without affecting the test's assertions in any way.